db/migrate/20210322225751_create_reminder_templates.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReminderTemplates < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reminder_templates do |t|
+      t.integer :experiment_group, null: false
+      t.string :message, null: true
+      t.integer :appointment_offset, null: true
+      t.references :reminder_experiment, null: false, foreign_key: true
+      t.timestamps
```

Comment:
`null: false`

Reply:
I'll add that to all of these. Looking at our current migrations, very few of them enforce this constraint. Perhaps they should.